src/Microsoft.Azure.ServiceBus/Primitives/SharedAccessSignatureTokenProvider.cs

```diff
@@ -104,39 +132,34 @@ static class SharedAccessSignatureBuilder
                 string targetUri,
                 TimeSpan timeToLive)
             {
-                // Note that target URI is not normalized because in IoT scenario it
+                // Note that target URI is not normalized because in IoT scenario it 
                 // is case sensitive.
                 string expiresOn = BuildExpiresOn(timeToLive);
                 string audienceUri = WebUtility.UrlEncode(targetUri);
-                var fields = new List<string> { audienceUri, expiresOn };
+                List<string> fields = new List<string> { audienceUri, expiresOn };
```

Comment:
nitpicking - any reason to switch from `var` to explicit type?

Reply:
Class is copied directly from EventHubs library. Not intentional. Please omit them.

---
In reply to: [156530826](https://github.com/Azure/azure-service-bus-dotnet/pull/385#discussion_r156530826) [](ancestors = 156530826)